herring/puzzles/templatetags/to_json.py

```diff
@@ -0,0 +1,11 @@
+import json
+
+from django import template
+
+from puzzles.models import to_json_value
+
+register = template.Library()
+
+@register.filter(name='to_json')
+def to_json(field):
+    return json.dumps(to_json_value(field))
```

Comment:
This looks like it only handles a single field. But it's used in the template to filter an entire object?